Makefile

```diff
@@ -32,7 +32,12 @@ fmt:
 	@echo gofmt
 
 image:
-	podman build  -f Dockerfile.rhel8 -t $(LOCAL_IMAGE_TAG) .
+	buildah build --platform=$(TARGET_PLATFORMS) --manifest eventrouter -f Dockerfile.rhel8
```

Comment:
why switch to buildah directly instead of podman?

Reply:
Actually I was wondering why the project changed from docker to podman few months ago 😂
That's the suggested way of podman: https://podman.io/blogs/2021/10/11/multiarch.html

podman can be launched with all the architectures as a parameter but the manifest will contain only the last processed. It's as you wish, I can do a for loop, invoking podman for earch architecture and add to the manifest using `podman manifest`.

Let me know what you prefer :-)